cli/utilities/packages.py

```diff
@@ -101,6 +115,16 @@ def clean(self):
         if self.execute(sudo=True, long_running=True, cmd=cmd):
             raise PackageError("Failed to clean repositories")
 
+    @staticmethod
```

Comment:
This method can be utilised in many places, better to move it some common place.

Reply:
will move to utils